Bottom: 12be089eab1fc3efb16038d1df1d560b204eaf44
Top:    cff43a0fdece56b694d4b86329248c3372f380cd
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2013-12-31 23:32:48 -0500

Refresh of zarlink

---

diff --git a/package/kernel/linux/modules/sound.mk b/package/kernel/linux/modules/sound.mk
index 2bc3ed1..a248d78 100644
--- a/package/kernel/linux/modules/sound.mk
+++ b/package/kernel/linux/modules/sound.mk
@@ -288,7 +288,7 @@ define KernelPackage/sound-soc-rt5350
   FILES:= \
 	$(LINUX_DIR)/sound/soc/fsl/snd-soc-rt5350-pcm.ko
   AUTOLOAD:=$(call AutoLoad,56,snd-soc-rt5350-pcm)
-  DEPENDS:=@TARGET_rt5350 +kmod-sound-soc-core
+  DEPENDS:=@TARGET_rampips +kmod-sound-soc-core
   $(call AddDepends/sound)
 endef
 
@@ -306,7 +306,7 @@ define KernelPackage/sound-soc-rt5350-le89156
 	$(LINUX_DIR)/sound/soc/codecs/snd-soc-le89156.ko \
 	$(LINUX_DIR)/sound/soc/codecs/snd-soc-rt5350-le89156.ko
   AUTOLOAD:=$(call AutoLoad,57,snd-soc-rt5350-le89156)
-  DEPENDS:=@TARGET_rt5350 +kmod-snd-soc-rt5350
+  DEPENDS:=@TARGET_ramips +kmod-snd-soc-rt5350
   $(call AddDepends/sound)
 endef
